tests/tracer/test_trace_utils.py

```diff
@@ -297,29 +299,71 @@ def test_ext_service(int_config, pin, config_val, default, expected):
     assert trace_utils.ext_service(pin, int_config.myint, default) == expected
 
 
+@pytest.mark.parametrize("appsec_enabled", [False, True])
 @pytest.mark.parametrize(
-    "method,url,status_code,status_msg,query,request_headers",
+    "method,url,status_code,status_msg,query,request_headers,response_headers,uri,path_params,cookies",
     [
-        ("GET", "http://localhost/", 0, None, None, None),
-        ("GET", "http://localhost/", 200, "OK", None, None),
-        (None, None, None, None, None, None),
-        ("GET", "http://localhost/", 200, "OK", None, {"my-header": "value1"}),
-        ("GET", "http://localhost/", 200, "OK", "search?q=test+query", {"my-header": "value1"}),
+        ("GET", "http://localhost/", 0, None, None, None, None, None, None, None),
+        ("GET", "http://localhost/", 200, "OK", None, None, None, None, None, None),
+        (None, None, None, None, None, None, None, None, None, None),
+        (
+            "GET",
+            "http://localhost/",
+            200,
+            "OK",
+            None,
+            {"my-header": "value1"},
+            {"resp-header": "val"},
+            "http://localhost/",
+            None,
+            None,
+        ),
+        (
+            "GET",
+            "http://localhost/",
+            200,
+            "OK",
+            "q=test+query&q2=val",
+            {"my-header": "value1"},
+            {"resp-header": "val"},
+            "http://localhost/search?q=test+query&q2=val",
+            {"/search": lambda x: x},
```

Comment:
?

Reply:
Sorry, let me add context.... I am not sure what this means, why is this a lambda? If this is path params, shouldn't it be something like `{"id": "1234"}` ?